apps/db_pendings/import.sh

```diff
@@ -65,14 +64,13 @@ function import() {
                 echo "DELIMITER //"  >> "$newFile";
                 echo "CREATE PROCEDURE updateDb ()" >> "$newFile";
                 echo "proc:BEGIN DECLARE OK VARCHAR(100) DEFAULT 'FALSE';" >> "$newFile";
-                echo "SELECT COUNT(*) INTO @COLEXISTS"  >> "$newFile";
-                echo "FROM information_schema.COLUMNS" >> "$newFile";
-                echo "WHERE TABLE_SCHEMA = DATABASE() AND TABLE_NAME = 'version_db_"$db"' AND COLUMN_NAME = '"$oldVer"';" >> "$newFile";
-                echo "IF @COLEXISTS = 0 THEN LEAVE proc; END IF;" >> "$newFile";
+                echo "SELECT date INTO @COLEXISTS" >> "$newFile";
+                echo "FROM version_db_"$db  >> "$newFile";
+                echo "ORDER BY date DESC LIMIT 1;"  >> "$newFile";
+                echo "IF @COLEXISTS <> '"$oldVer"' THEN LEAVE proc; END IF;" >> "$newFile";
             fi
 
             echo "$startTransaction" >> "$newFile";
-            echo "$updHeader" >> "$newFile";
```

Comment:
If you prefer I can revert this and only have the update part for now... for maybe faster merge :shrug: